pkg/operator/resource/resourceapply/apiext.go

```diff
@@ -0,0 +1,34 @@
+package resourceapply
```

Comment:
no abbreviations in the filename please. `apiextensions.go`

Reply:
fixed